Example/package.json

```diff
@@ -6,8 +6,12 @@
     "start": "node node_modules/react-native/local-cli/cli.js start"
   },
   "dependencies": {
-    "react": "~15.4.0",
+    "react": "^15.4.0",
```

Comment:
>^ [](start = 14, length = 1)

Reset.

Reply:
Hi...Can you please share the steps to integrate this library in react-native-windows?

Reply:
https://shift.infinite.red/getting-started-with-react-native-development-for-windows-ba23a4be90b6
 check it out!